public/data/Jatinkumar30.json

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "Jatin Kumar",
+    "type": "personal",
+    "bio": "Open Source Enthusiast and Front end developer!",
+    "avatar": "https://github.com/jatinkumar30.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/jatinkumar30",
```

Comment:
```suggestion
        "url": "https://github.com/Jatinkumar30",
```

Reply:
Updated github url @Panquesito7 please do check it out.